src/models/task/model.py

```diff
@@ -0,0 +1,147 @@
+class TaskOrientedChatbot:
+    def __init__(self):
+        self.done = False
+        self.collected_data = self._init_data()
+        self.stage = 0
+        "possible stages:\n"
+        "        0. ask to confirm task start\n"
+        "        1. confirm start\n"
+        "        2. ask for information\n"
+        "           provide answer\n"
+        "        3. confirm end\n"
+        "           ask for rechecking\n"
+        "        4. confirm rechecking\n"
+        "        5. recheck information\n"
+        self.current_field = None
+
+
+    def _is_completed(self):
+        return self.done
+
+    @staticmethod
+    def _init_data():
+        return dict(
+            group=dict(
+                data_type=int,
+                value=None,
+                question="O którą grupę pytasz?",
+                priority=1,
+                check=False
+            ),
+            tutor=dict(
+                data_type=int,
+                value=None,
+                question="Kto prowadzi zajęcia?",
+                priority=1,
+                check=False
+            ),
+            course=dict(
+                data_type=int,
+                value=None,
+                question="O jaki przedmiot pytasz?",
+                priority=1,
+                check=False
+            ),
+            room=dict(
+                data_type=int,
+                value=None,
+                question="W jakiej sali odbywają się zajęcia?",
+                priority=1,
+                check=False
+            ),
+            start_hour=dict(
+                data_type=int,
+                value=None,
+                question="O której rozpoczynają się zającia?",
+                priority=1,
+                check=False
+            )
+        )
+
+    def _detect_confirmation(self, prompt):
```

Comment:
We should change the prompt to lowercase. Otherwise `Tak` is not going to be registered as confirmation.